sydent/http/servlets/internalinfoservlet.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+
+# copyright 2019 new vector ltd
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+
+from twisted.web.resource import Resource
+
+import logging
+import json
+
+from sydent.db.invite_tokens import JoinTokenStore
+from sydent.http.servlets import get_args, jsonwrap, send_cors
+from sydent.http.info import Info
+
+
+logger = logging.getLogger(__name__)
+
+
+class InternalInfoServlet(Resource):
+    isLeaf = True
+
+    def __init__(self, syd):
+        self.sydent = syd
+        self.info = Info(syd)
```

Comment:
you only need one of these, not one per servlet.

> which you instantiate before the servlets
> and then pass that instance into the servlet constructors

(ie, instantiate it in sydent.py)